gravitee-am-identityprovider/gravitee-am-identityprovider-api/src/main/java/io/gravitee/am/identityprovider/api/oidc/OpenIDConnectIdentityProviderConfiguration.java

```diff
@@ -30,4 +31,5 @@
     public SignatureAlgorithm getSignatureAlgorithm();
     public String getResolverParameter();
     public boolean isEncodeRedirectUri();
+    ClientAuthentication getClientAuthentication();
```

Comment:
I prefer using a default method : 
```
String getClientAuthenticationMethod() {
  return ClientAuthenticationMethod.CLIENT_SECRET_POST;
}
```

This way no need to update all the providers and we are still compliant with possible custom IdPs.